TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8260,7 +8260,9 @@ execStatementText: ExecStatementText;
 execStatementEnd: END_EXEC;
 
 //FOR SQL
-commitStatement: SQL_COMMIT;
+// Defining 'WORK' as contextual keyword here since it is not part of reserved words
+commitStatement: SQL_COMMIT work?;
+work: ({ string.Equals(CurrentToken.Text, "WORK", System.StringComparison.OrdinalIgnoreCase) }? KeywordWORK=UserDefinedWord);
```

Comment:
@laabidiHend Can you please add a test with a COMMIT statement wich is followed by another identifier than WORK. Just to check that a syntax error is reported.